controller/api/views.py

```diff
@@ -207,7 +207,8 @@ def logs(self, request, **kwargs):
         try:
             return Response(app.logs(request.query_params.get('log_lines',
                                      str(settings.LOG_LINES))),
-                            status=status.HTTP_200_OK, content_type='text/plain')
+                            status=status.HTTP_200_OK,
```

Comment:
We should avoid the temptation to apply formatting fixes in this sort of PR.  If formatting fixes to existing code are needed, they should be applied in their own separate chore PR.